lib/bbce-api/models/bilateral_company_deadline_value_payload.rb

```diff
@@ -284,7 +280,7 @@ def to_body
     # Returns the object in the form of hash
     # @return [Hash] Returns the object in the form of hash
     def to_hash
-      hash = super
+      hash = {}
```

Comment:
Perhaps change this to `Hash.new` to improve consistency.
```suggestion
      hash = Hash.new
```